arch/X86/X86Mapping.c

```diff
@@ -3467,13 +3467,13 @@ void op_addAvxBroadcast(MCInst *MI, x86_avx_bcast v)
 #ifndef CAPSTONE_DIET
 // map instruction to its characteristics
 typedef struct insn_op {
-	uint64_t eflags;	// how this instruction update EFLAGS
+	uint64_t flags;	// how this instruction update EFLAGS
```

Comment:
the comment on this line should be "how this instruction update EFLAGS (for arithmetic instructions), or FPU FLAGS (for FPU instructions)